cms/djangoapps/contentstore/views/tests/test_clipboard_paste.py

```diff
@@ -477,6 +477,16 @@ def test_paste_from_library_read_only_tags(self):
             assert object_tag.value in self.lib_block_tags
             assert object_tag.is_copied
 
+        # If we delete the upstream library block...
+        library_api.delete_library_block(self.lib_block_key)
+
+        # ...the copied tags remain, but should no longer be marked as "copied"
+        object_tags = tagging_api.get_object_tags(new_block_key)
+        assert len(object_tags) == len(self.lib_block_tags)
+        for object_tag in object_tags:
+            assert object_tag.value in self.lib_block_tags
+            assert not object_tag.is_copied
+
```

Comment:
Since we depend on openedx-events for this to work, it would be safe to explicitly enable the required events to make sure that the test passes (had an issue before with a test case that failed only in CI but passed locally, most probably due to some events being disabled in some previous test in CI).

You can inherit from `OpenEdxEventsTestMixin` (Note that it should be added before `ModuleStoreTestCase`) and enable below events.

```python
class ClipboardPasteFromV2LibraryTestCase(OpenEdxEventsTestMixin, ModuleStoreTestCase):
    """
    Test Clipboard Paste functionality with a "new" (as of Sumac) library
    """

    ENABLED_OPENEDX_EVENTS = [
        LIBRARY_BLOCK_DELETED.event_type,
        XBLOCK_UPDATED.event_type,
    ]
```